src/app/modules/calls/calls.service.ts

```diff
@@ -1,16 +1,49 @@
-import { PrismaService } from 'src/libraries/prisma/prisma.service';
+import { PrismaService } from 'src/app/libraries/prisma/prisma.service';
 import { Call } from '@prisma/client';
 import { PinoLogger } from 'nestjs-pino';
 import { CreateCallInput, EndCallInput } from './calls.schemas';
 import { Injectable } from '@nestjs/common';
+import { InjectQueue } from '@nestjs/bullmq';
+import { Queue } from 'bullmq';
 
 @Injectable()
 export class CallsService {
   constructor(
     private readonly prisma: PrismaService,
     private readonly logger: PinoLogger,
+    @InjectQueue('callsWebhooks') private readonly callsWebhooksQueue: Queue,
   ) {}
 
+  async addCallToCreateToQueue(
+    call: CreateCallInput,
+  ): Promise<{ success: boolean }> {
+    try {
+      const job = await this.callsWebhooksQueue.add('processCreateCall', call);
+      if (job?.id) {
+        return { success: true };
+      } else {
+        return { success: false };
+      }
```

Comment:
If job is enqueued, we call success to the endpoint, which calls this service method